CHANGES.md

```diff
@@ -1,3 +1,19 @@
+Unreleased
+-----
+  * **BREAKING:** Rewrite `ppx_import` as a context free rule. It changes the syntax of the type declaration from:
+
+  ```ocaml
+  type loc = [%import: Location.t];;
+  ```
+
+  to:
+
+  ```ocaml
+  [%%import : type loc = Location.t]
+  (* or *)
+  type%import loc = Location.t
+  ```
+
```

Comment:
Would it be possible to explain how the other parts of the syntax work? In particular the `with foo := bar` alias.

Maybe it is better just to update the readme in this PR too? Indeed, it could be a good idea to keep the documentation of the code in sync.